aws/resource_aws_instance_test.go

```diff
@@ -2780,6 +2872,198 @@ resource "aws_instance" "foo" {
 }
 `
 
+const testAccCheckInstanceConfigWithAttachedEbs = `
```

Comment:
It looks like there's a fair amount of duplication with these new configs - it's ok to reuse the same config for multiple tests, so if there's any way to reuse the config (or create a new `func testAccInstanceConfig...` function with variable input as needed) that would help cut down on the duplication!

Reply:
Caveat:  I'm not a developer, I just pretend to be one from time to time when a software issue annoys me enough to do something about it.  With that said...

I've addressed the first 5 comments with the latest commit, as they were rather trivial.

I may have botched the rebase, as all the commits/files changed, since I initially submitted this PR, seem to be appearing as part of this PR now.  Not quite sure how to go about sorting that out or if that is expected.

I understand your point concerning config duplication and was looking at how functions were used in other test code.  It is not immediately obvious (to me) how to go about using a function, in this context, for better reuse, but I will certainly figure it out.  I am particularly pressed for time this week and may not be able to revisit this until next weekend.

Any further questions/comments/suggestions are certainly welcome.  Thanks.

Reply:
Caveat: I'm not a developer, I just pretend to be one from time to time when a software issue annoys me enough to do something about it. With that said...

I've addressed the first 5 comments with the latest commit, as they were rather trivial.

I may have botched the rebase, as all the commits/files changed, since I initially submitted this PR, seem to be appearing as part of this PR now. Not quite sure how to go about sorting that out or if that is expected.

I understand your point concerning config duplication and was looking at how functions were used in other test code. It is not immediately obvious (to me) how to go about using a function, in this context, for better reuse, but I will certainly figure it out. I am particularly pressed for time this week and may not be able to revisit this until next weekend.

Any further questions/comments/suggestions are certainly welcome. Thanks.

Reply:
@sigsegv13 Try running `git pull --rebase origin master` (assuming your `origin` git remote is pointed to the terraform-providers repo and not your fork. use `git remote -vv` to confirm or find out the remote name)

That should update your branch so that it doesn't have all of the unrelated commits that are also in the master branch. `git push --force` your branch and then this PR should have only the relevant commits in it.

Reply:
@rifelpet Got it sorted out.  Thanks.